components/docs-chef-io/content/automate/ha_chef_backend_to_automate_ha.md

```diff
@@ -17,14 +17,15 @@ gh_repo = "automate"
 {{< /warning >}}
 
 {{< warning >}}
-- Customers using only **Chef Backend** are advised to follow this migration guidance. Customers using **Chef Manage** or **Private Chef Supermarket** with Chef Backend should not migrate with this.
+- Customers using only **Standalone Chef Infra Server** or **Chef Backend** are advised to follow this migration guidance. Customers using **Chef Manage** or **Private Chef Supermarket** with Chef Backend should not migrate with this.
+- Also for the customers using standalone Chef Infra Server, cookbooks should be in the database but not in either the file system or S3.
```

Comment:
- Also, for the customers using a standalone Chef Infra Server, cookbooks should be in the database but not in either the file system or S3.